pkgs/development/mobile/androidenv/emulate-app.nix

```diff
@@ -1,16 +1,19 @@
 { composeAndroidPackages, stdenv, lib, runtimeShell }:
-{ name, app ? null
+{ name
+, app ? null
 , platformVersion ? "33"
 , abiVersion ? "armeabi-v7a"
 , systemImageType ? "default"
-, enableGPU ? false
-, extraAVDFiles ? []
+, enableGPU ? false # Enable GPU acceleration
+, configOptions ? lib.optional enableGPU [ "hw.gpu.enabled=yes" ] # List of options to add in config.ini
```

Comment:
Shouldn't `lib.optional` be `lib.optionals`?

Reply:
@badly-drawn-wizards thanks for the review. it's fixed in https://github.com/NixOS/nixpkgs/pull/265504/commits/b8e1c8203cb7fe0b13de3767b021b9f4622b3eca